.flake8

```diff
@@ -227,3 +267,42 @@ per-file-ignores =
   tests/unit/image_manager/__init__.py: D104
   tests/unit/__init__.py: D104
   share/ansible_navigator/utils/__init__.py: D104
+
+# Count the number of occurrences of each error/warning code and print a report:
+statistics = true
+
+# flake8-eradicate
+# E800:
+eradicate-whitelist-extend = isort:\s+\w+
```

Comment:
Since we are not using this, can this be removed?

Reply:
I'd rather keep it for the sake of having the configs similar. Also, this plugin is a part of WPS. So technically we are using it. We don't have isort enabled, that is true. But my feeling is that we may enable it at some point and if we end up needing its marker comments, they'll likely produce false positives in `erradicate`. Since this problem has been debugged and sorted with this config in other projects, I don't see why we shouldn't include it and face this false-positive that would definitely confuse the contributors and waste somebody's debug time.

Reply:
I understand, can we note that it's existence in the file for forward looking to a future implementation and not in effect today?  If not in this PR, can you log an issue for followup?

Reply:
https://github.com/ansible/ansible-navigator/issues/723